django/contrib/gis/gdal/geometries.py

```diff
@@ -268,6 +275,34 @@ def extent(self):
         "Return the envelope as a 4-tuple, instead of as an Envelope object."
         return self.envelope.tuple
 
+    @property
+    def is_3d(self):
+        """Return True if the geometry has Z coordinates."""
+        return capi.is_3d(self.ptr)
+
+    def set_3d(self, value):
+        if value is True:
+            capi.set_3d(self.ptr, 1)
+        elif value is False:
+            capi.set_3d(self.ptr, 0)
+        else:
+            raise ValueError(f"Input to 'set_3d' must be a boolean, got {value!r}.")
```

Comment:
What do you think about adding `is_3d` and `set_3d` in a separate PR? We could merge it first.

Reply:
I can move it if you want.

Reply:
I'm happy to do this, the tests will need a bit of work.

Likely we can do the same with the change to use the `Iso` output for wkt/wkb.

Reply:
Sounds good :+1: Thanks.

Reply:
Merged, please rebase.